packages/select-rich/src/LionSelectRich.js

```diff
@@ -38,6 +38,17 @@ function isInView(container, element, partial = false) {
   return isTotal || isPartial;
 }
 
+function handleKeydown(event) {
+  switch (event.key) {
+    case 'ArrowDown':
+    case 'ArrowUp':
+    case 'Home':
```

Comment:
I would argue Home and End keys should not be prevent defaulted here. They have no meaningful interaction on the invoker, only if you are inside the listbox. And on the listbox node those keys are already prevent defaulted. Here on the invoker they should act as normal and scroll the user down right?

Reply:
Yes it's working when moved in the invoker